server/routes/api.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express')
+const City = require('../model/City')
+const { formatObject, 
+        getCityImage,
+        getCityWeatherByLatLon,
+        getCityWeatherByName, getCitiesByIds,
+        getCityWeatherById, checkLastUpdatedAndRefresh
+      } = require('../helpers/functions')
+const router = express.Router()
+
+router.get('/sanity', (req,res) => {
+  res.send('OK')
+})
+
+router.get('/city/:cityName', async (req,res) => {
+  const {cityName} = req.params
+  try {
+    let cityWeather = await getCityWeatherByName(cityName)                                                           
+    let cityImage = await getCityImage(cityName)
+    let formatedObject = formatObject(cityWeather, cityImage)
+    res.send(formatedObject)
+  } catch(err) {
+    console.error(err)
+    res.status(500)
+    res.send({error: "Error recieving city data"})
+  }
+})
+
+router.get('/city/:lat/:lon', async (req,res) => {
+  const {lat, lon} = req.params
+  try {
+    let cityWeather = await getCityWeatherByLatLon(lat, lon) 
+    let cityImage = await getCityImage(cityWeather.name)
+    let formatedObject = formatObject(cityWeather,cityImage)
+    res.send(formatedObject)
+  } catch(err) {
+    console.error(err)
+    res.status(500)
+    res.send({error: "Error recieving city by location data"})
+  }
+})
+
+router.post('/city', async (req, res) => {
+  const {name, temp, condition, feelsLike, icon, cityId, lastUpdate, cityImage} = req.body
+  try {
+    const cityToSave = new City({name, temp, condition, feelsLike, icon, cityId, lastUpdate, cityImage})
```

Comment:
you could just do - 
const cityToSave  = new City(req.body)

instead of restructuring everything...